js/tabRestore.js

```diff
@@ -0,0 +1,33 @@
+const MAX_TAB_HISTORY_DEPTH = 20
+var tabHistoryStack = Array()
```

Comment:
Oh, I see what you mean now. You're right, `tabHistory` should be private as there is room for misuse here.